nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -141,9 +147,9 @@ in
     (mkRenamedOptionModule
       [ "virtualisation" "libvirtd" "qemuOvmf" ]
       [ "virtualisation" "libvirtd" "qemu" "ovmf" "enable" ])
-    (mkRenamedOptionModule
+    (mkRemovedOptionModule
       [ "virtualisation" "libvirtd" "qemuOvmfPackage" ]
-      [ "virtualisation" "libvirtd" "qemu" "ovmf" "package" ])
+      "If this option was set to `foo`, set the option `virtualisation.libvirtd.qemu.ovmf.packages' to `[foo.fd]` instead.")
```

Comment:
Not sure about this line/change.

Reply:
I don't know what you find unclear, so maybe you can suggest a formulation that would be clearer to you?

The point is that if before you had `virtualisation.libvirtd.qemuOvmfPackage = foo` you should replace that by `virtualisation.libvirtd.qemu.ovmf.packages = [ foo.fd ]`.

Reply:
ah, it will just show that as an error message. That sounds ok to me.